### PR TITLE
Prod hotfix 1

### PR DESCRIPTION
**Pull request by EdPutans**

### CODE REVIEW

File: src/context/CollectorContext.tsx [hunk 1]
```diff
@@ -94,94 +117,74 @@ export const CollectorProvider = ({
     )
 
     if (!trigger) {
-      error('No trigger found for displayTrigger', displayTrigger)
+      error(`No trigger found for displayTrigger`, displayTrigger)
```
Comment (blixenkrone) [inline src/context/CollectorContext.tsx:120]: No string interpolation done?

**Review by blixenkrone — APPROVED** [review]

File: src/context/CollectorContext.tsx [hunk 2]
```diff
@@ -17,16 +18,10 @@ export type CollectorProviderProps = {
...
 }
 
-type DisplayTrigger =
-  | 'INVOCATION_UNSPECIFIED'
-  | 'INVOCATION_IDLE_TIME'
-  | 'INVOCATION_EXIT_INTENT'
-  | 'INVOCATION_PAGE_LOAD'
-
-export const CollectorProvider = ({
+export function CollectorProvider({
```
Comment (AdenFraser) [inline src/context/CollectorContext.tsx:21]: Why the change in syntax?

File: src/context/CollectorContext.tsx [hunk 3]
```diff
@@ -70,20 +73,40 @@ export const CollectorProvider = ({
...
-
-  const showTrigger = (displayTrigger: DisplayTrigger | undefined) => {
-    if (!displayTrigger) {
-      return null
+  }, [intently, log])
+
+  const TriggerComponent = React.useCallback(() => {
+    if (!displayTrigger) return null
+
+    // TODO: UNDO
```
Comment (AdenFraser) [inline src/context/CollectorContext.tsx:81]: It would be good when adding todo's if you could use the `@todo` syntax — IDEs will then pick these up etc :)

**Review by AdenFraser — APPROVED** [review]
Nothing here blocking if this is going to fix prod

File: src/context/CollectorContext.tsx (hunk 2 above)
Comment (EdPutans) [reply src/context/CollectorContext.tsx:21]: cause I ran it through ESLINT to get all the hook deps sorted

File: src/context/CollectorContext.tsx (hunk 3 above)
Comment (EdPutans) [reply src/context/CollectorContext.tsx:81]: image
🤷‍♀️

File: src/context/CollectorContext.tsx (hunk 1 above)
Comment (EdPutans) [reply src/context/CollectorContext.tsx:120]: use whichever you like, Simon, you're a free spirit
